src/connector.ts

```diff
@@ -134,4 +142,20 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
+			const RUNDOWN_EXTERNAL_ID_SUFFIX = '_1'
+			if (!this.iNewsFTPHandler.iNewsWatcher) {
+				return
+			}
+			this.iNewsFTPHandler.iNewsWatcher?.ResyncRundown(ctx.params.rundownName + RUNDOWN_EXTERNAL_ID_SUFFIX)
+
+			await next()
+		})
+
+		this.koaApp.use(this.koaRouter.routes()).use(this.koaRouter.allowedMethods())
+
+		this.koaApp.listen(3007, () => {})
```

Comment:
Would be nice if the port was also in a constant.

Reply:
Moved into a constant `KOA_PORT`.